web/client/actions/widgets.js

```diff
@@ -248,6 +251,29 @@ const toggleDependencySelector = (active, settings) => setupDependencySelector({
     active,
     ...settings
 });
+/**
+ * Collapse/Expand the widget
+ * @param {object} widget the widget to collapse
+ */
+const toggleCollapse = (widget, target = DEFAULT_TARGET) => ({
```

Comment:
test for this, i know it is an action but you choose

Reply:
tested with the reducer, anyway, I need to add a test for the action itself